sys/include/net/netopt.h

```diff
@@ -269,6 +269,28 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /**
+     * @brief LoRA options.
+     */
+    NETOPT_LORA_BANDWIDTH,                        /**< bandwidth of LoRa */
```

Comment:
Other radios have bandwidths, spreading factors, coding rates, etc., too. So think hard if all these options really need to be LoRa specific.